models/AmazonS3.cfc

```diff
@@ -54,23 +54,30 @@ component accessors="true" singleton {
      */
     public AmazonS3 function init(
         required string accessKey,
-        required string secretKey,
+		required string secretKey,
+		string awsRegion = "us-east-1",
+		string awsDomain = "amazonaws.com",
+		string awsServiceName = "s3",
         string encryption_charset = "UTF-8",
-        boolean ssl = false,
-        string awsRegion = "us-east-1"
+        boolean ssl = true
+
     ) {
         variables.accessKey = arguments.accessKey;
         variables.secretKey = arguments.secretKey;
         variables.encryption_charset = arguments.encryption_charset;
-        variables.awsRegion = arguments.awsRegion;
+		variables.awsRegion = arguments.awsRegion;
+		variables.awsDomain = arguments.awsDomain;
+		variables.awsServiceName = arguments.awsServiceName;
+
+
         setSSL( ssl );
 
         variables.sv4Util = new Sv4Util(
             accessKeyId = variables.accessKey,
             secretAccessKey = variables.secretKey,
             defaultRegionName = arguments.awsRegion,
-            defaultServiceName = "s3"
-        );
+            defaultServiceName = 's3'
```

Comment:
Where is the change to this util. I don’t see it?

Reply:
Just single quotes. Probably shouldn't have been pushed.

Reply:
it was just single quotes around s3, probably should have been excluded from commit

Reply:
NO, what I mean, is that the default Service Name argument is not in the sv4Util code? Where is it?